src/components/TextStatsItem.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TextStatsItem = ({ name = '', value = '' }) => (
+  <div className='text-stats__item'>
+    <p className='text-stats__name'>{name}</p>
+    <p>{value}</p>
+  </div>
+);
+
+TextStatsItem.propTypes = {
+  name: PropTypes.string,
+  value: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+    PropTypes.object,
```

Comment:
If it's an object we shouldn't write it out – is this a fallback?

Reply:
this has to do with the implementation of the BigNumber component I'm using here – it's an object BUT it returns a string :wink: